include/dlaf/lapack/enum_output.h

```diff
@@ -0,0 +1,40 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2019, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+
+#pragma once
+
+#include <ostream>
+
+#include <lapack_util.hh>
+
+namespace lapack {
+
+inline std::ostream& operator<<(std::ostream& out, const Norm& norm) {
+  switch (norm) {
+    case lapack::Norm::One:
+      out << "One";
```

Comment:
Wondering if it is not better to specify Norm in the output, otherwise the string might be misleading. E.g. `OneNorm` or NormOne`